kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/powah/energizing.js

```diff
@@ -10,77 +11,13 @@ onEvent('recipes', (event) => {
             result: {
                 item: 'tetra:magmatic_cell',
                 count: 1
-            }
-        },
-        {
-            ingredients: [{ tag: 'forge:storage_blocks/iron_copper' }, { tag: 'forge:storage_blocks/gold' }],
-            energy: '100000',
-            result: {
-                item: 'powah:energized_steel_block',
-                count: 2
-            }
-        },
-        {
-            ingredients: [{ item: 'botania:blaze_block' }],
-            energy: '900000',
-            result: {
-                item: 'powah:blazing_crystal_block',
-                count: 1
-            }
-        },
-        {
-            ingredients: [{ tag: 'forge:storage_blocks/diamond' }],
-            energy: '3000000',
-            result: {
-                item: 'powah:niotic_crystal_block',
-                count: 1
-            }
-        },
-        {
-            ingredients: [{ tag: 'forge:storage_blocks/emerald' }],
-            energy: '10000000',
-            result: {
-                item: 'powah:spirited_crystal_block',
-                count: 1
-            }
+            },
+            id: `${id_prefix}magmatic_cell`
         }
     ];
 
     recipes.forEach((recipe) => {
-        event.custom({
-            type: 'powah:energizing',
-            ingredients: recipe.ingredients,
-            energy: recipe.energy,
-            result: recipe.result
-        });
+        recipe.type = 'powah:energizing';
+        event.custom(recipe).id(recipe.id);

```

Comment:
It does :D have not encountered any issues with it so far. Not sure if it could cause issues in the future, but my understanding is the recipe deserializers are looking for specific entries and anything else is ignored.

Reply:
Sweeeet!